test/arcade/core/util/ColorsTest.java

```diff
@@ -0,0 +1,135 @@
+
+import java.awt.Color;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ColorsTest {
+
+    @Test
+    public void getColor_calledIndices_returnsColors() {
+        Color[] colorArray = new Color[] {new Color(1, 1, 1, 1), new Color(255, 255, 255, 1)};
+        Colors colors = new Colors(colorArray);
+        assertEquals(colorArray[0], colors.getColor(0));
+        assertEquals(colorArray[1], colors.getColor(1));
+    }
+
+    @Test
+    public void getRGB_calledIndices_returnsRGB() {
+        Color[] colorArray = new Color[] {new Color(1, 1, 1, 1), new Color(255, 255, 255, 1)};
+        Colors colors = new Colors(colorArray);
+        assertEquals(colorArray[0].getRGB(), colors.getRGB(0));
+        assertEquals(colorArray[1].getRGB(), colors.getRGB(1));
+    }
+
+    @Test
+    public void getColor_calledDoubles_returnsRoundedDownColor() {
+        Color[] colorArray = new Color[] {new Color(1, 1, 1, 1), new Color(255, 255, 255, 1)};
+        Colors colors = new Colors(colorArray);
+        assertEquals(colorArray[0], colors.getColor(0.5));
+        assertEquals(colorArray[0], colors.getColor(0.99));
+        assertEquals(colorArray[1], colors.getColor(1.1));
+    }
+
+    @Test
+    public void getRGB_calledDoubles_returnsRoundedDownColor() {
+        Color[] colorArray = new Color[] {new Color(1, 1, 1, 1), new Color(255, 255, 255, 1)};
+        Colors colors = new Colors(colorArray);
+        assertEquals(colorArray[0].getRGB(), colors.getRGB(0.5));
+        assertEquals(colorArray[0].getRGB(), colors.getRGB(0.99));
+        assertEquals(colorArray[1].getRGB(), colors.getRGB(1.1));
+    }
+
+    @Test
+    public void getAlpha_called_returnsValid() {
+        Color[] colorArray = new Color[] {new Color(1, 1, 1, 1), new Color(255, 255, 255, 1)};
+        Colors colors = new Colors(colorArray);
+        assertEquals(1, colors.getAlpha(0.0));
+    }
+
+    @Test
+    public void getColor_called_returnsValid() {
+        Colors colors = new Colors(new Color(1, 1, 1, 1), new Color(255, 255, 255, 1), 0.0, 1.0);
+        assertEquals(new Color(255, 255, 255, 1), colors.getColor(1.0));
+    }
+
+    @Test
+    public void getRGB_called_returnsValid() {
+        Colors colors = new Colors(new Color(1, 1, 1, 1), new Color(255, 255, 255, 1), 0.0, 1.0);
+        assertEquals(new Color(255, 255, 255, 1).getRGB(), colors.getRGB(1.0));
+        assertEquals(new Color(1, 1, 1, 1).getRGB(), colors.getRGB(0.0));
+    }
+
+    @Test
+    public void getColor_calledInvalidNumber_returnsEmptyColor() {
+        Colors colors = new Colors(new Color(1, 1, 1, 1), new Color(255, 255, 255, 1), 0.0, 1.0);
+        assertEquals(new Color(0, 0, 0, 0), colors.getColor(-1.0));
+        assertEquals(new Color(0, 0, 0, 0), colors.getColor(-1.5));
+        assertEquals(new Color(0, 0, 0, 0), colors.getColor(-2.0));
+    }
+
+    @Test
+    public void getColor_calledBetweenMinAndDefault_returnsMinColor() {
```

Comment:
```suggestion
    public void getColor_calledSmallerThanMin_returnsMinColor() {
```

Reply:
I am renaming the previous test where it's testing values smaller than default.